mne/epochs.py

```diff
@@ -1790,14 +1801,14 @@ def __init__(self, raw, events, event_id, tmin, tmax, baseline=(None, 0),
         proj = proj or raw.proj
 
         # call _BaseEpochs constructor
-        super(Epochs, self).__init__(info, None, events, event_id, tmin, tmax,
-                                     baseline=baseline, raw=raw, picks=picks,
-                                     name=name, reject=reject, flat=flat,
-                                     decim=decim, reject_tmin=reject_tmin,
-                                     reject_tmax=reject_tmax, detrend=detrend,
-                                     add_eeg_ref=add_eeg_ref, proj=proj,
-                                     on_missing=on_missing,
-                                     preload_at_end=preload, verbose=verbose)
+        super(Epochs, self).__init__(
```

Comment:
why did you change these lines? I don't see a difference except at the last line ...